src/CoffeeAddictsMain.kt

```diff
@@ -0,0 +1,44 @@
+import parser.FileParser
+import java.io.File
+
+/**
+ * created by AlexandraV on 12/25/2018
+ */
+fun main(args: Array<String>) {
+    if (!areCommandLineArgumentsValid(args)) {
+        return
+    } else {
+        // read the coordinate of the user
+        val userCoordinate = Coordinate(args[0], args[1])
+
+        // read content of .csv file
+        val fileParser = FileParser(args[2], userCoordinate)
+        fileParser.parseCoffeeShopsList(3)
```

Comment:
By the looks of the usage of this method, it as side effects and by passing in `3` as argument looks like does more things that it should. One would expect that `parseCoffeeShops` would process some kind of serialized data and return a list of coffee shops objects.